integration/test/src/docker/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM nginx
+
```

Comment:
Could we add some documentation in this Dockerfile to understand whats going on here?

Reply:
https://github.com/cerner/beadledom/pull/149/commits/00f709820b4beb51b556119aceb9ecd6bfdbe7c6